tests/foreman/cli/test_ansible.py

```diff
@@ -103,3 +104,50 @@ def test_positive_ansible_e2e(target_sat, module_org, rhel_contenthost):
     assert SELECTED_ROLE, (
         SELECTED_VAR not in target_sat.cli.Ansible.variables_info({'name': SELECTED_VAR}).stdout
     )
+
+
+def test_add_and_remove_ansible_role_hostgroup(target_sat):
```

Comment:
I would say this is a tier 2 association test. Could you add
```suggestion

@pytest.mark.tier2
def test_add_and_remove_ansible_role_hostgroup(target_sat):
```